ui/apps/platform/src/hooks/useFetchClustersForPermissions.ts

```diff
@@ -32,14 +31,8 @@ function useFetchClustersForPermissions(permissions: string[]): Result {
 
         getClustersForPermissions(requestedPermissions)
             .then((data) => {
-                const clusters: Cluster[] = data.clusters.map((responseCluster: ScopeObject) => {
```

Comment:
The response type for `getClustersForPermissions` is already `ScopeObject[]`, so this mapping is essentially a no-op.